api/views.py

```diff
@@ -89,68 +60,32 @@ def get_object(self):
     # def perform_create(self, serializer):
     #     serializer.save()
 
-#
-# class OrderListApiView(viewsets.ModelViewSet):
-#     queryset = Order.objects.all()
-#     serializer_class = OrderSerializer
-
 
 class OrderListApiView(viewsets.ModelViewSet):
-    queryset = Order.objects.all()
     serializer_class = OrderSerializer
 
     def get_queryset(self):
-        return self.queryset
+        return Order.objects.all()
```

Comment:
the same as above. If you're not using filtering by default, use `.none()` and use class argument `queryset`

Reply:
add pagination